src/containers/Sidebar/SidebarBottomItem.js

```diff
@@ -27,11 +24,6 @@ const SidebarBottomItem = ({ item, sidebarLogo }) => {
 		// Preserve from history pushing when item.url doesn't exist
 		// or when current location pathname is the same as item.url
 		if (item.url && location.pathname !== item.url) history.push(item.url);
-		// Preserve from collapsing when item doesn't have children
-		// or if item should always be uncollapsed
-		else if (item.children && !uncollapseAll) setOpen(prev => !prev);
-		// Close small sidebar on item click
-		if (isSmallSidebarOpen && window.innerWidth < 768) dispatch({ type: SET_SMALL_SIDEBAR });
 	}
```

Comment:
I also removed this condition `else if (item.children && !uncollapseAll) setOpen(prev => !prev);`, because we do not pass these parameters as props

Reply:
@Aringoaway Alright, then remove it also from here, if its not used: https://github.com/TeskaLabs/asab-webui/blob/da1b44afc9ae2328aa05adae6190769b30e0dcc7/src/containers/Sidebar/SidebarItem.js#L47